### PR TITLE
Feature/integrate header

### PR DESCRIPTION
**Pull request by archaengel**
Integrate header to display above all pages except for splash page, login, register, and password recovery page.

### CODE REVIEW

File: src/components/Toolbar/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <div className="spacer" />
+        <div className="toolbar__navigation-items">
+          <ul>
+            <li>
+              <a href="/login">login </a>
+            </li>
+            <li>
+              <a href="/register">register </a>
+            </li>
+            <form action="/jobs">
```
Comment (zihaozheng6) [inline src/components/Toolbar/index.js:35]: I think i have made a mistake here, the button of 'post a job' should route to '/dashboard' not '/jobs'.

**Review by zihaozheng6 — COMMENTED** [review]
Awesome！

**Review by suyogsathe7 — APPROVED** [review]
looks good.

**Review by ksharish72 — APPROVED** [review]
Looks good Edward! Moving forward, Please download a Code Formatter like Prettier and format document so that the formatting changes are not visible in commit differences and only the exact code changes are visible.

File: src/components/Toolbar/index.js (hunk 1 above)
Comment (archaengel) [reply src/components/Toolbar/index.js:35]: Good catch, I'll take care of it.

**Comment by archaengel** [comment]
@ksharish72 Thanks Harish! These changes were introduced by Prettier. Is there a way to configure Prettier to format code without changing it?

**Comment by ksharish72** [comment]
@archaengel I use the default format of prettier. I didnt configure it. Let check it out in our future commits.

**Comment by archaengel** [comment]
@ksharish72 Sounds good! It may be worth discussing committing a prettier config file to the repo to promote a consistent style! I'll switch to the default for this repo for now.

**Pull request merged**